components/devtools/actors/network_event.rs

```diff
@@ -300,8 +300,8 @@ impl Actor for NetworkEventActor {
                 let timings_obj = Timings {
                     blocked: 0,
                     dns: 0,
-                    connect: self.request.connect_time,
-                    send: self.request.send_time,
+                    connect: self.request.connect_time.as_nanos() as u64,
```

Comment:
Should we use `as_millis()` here? AFAICT, request.connect_time used to represent duration in milliseconds because its value is ultimately derived from https://github.com/servo/servo/blob/75c7712905c522f604a454dcb1c18b8e3f163d3b/components/net/http_loader.rs#L675

and connect_start and connect_end were intialized with

https://github.com/servo/servo/blob/75c7712905c522f604a454dcb1c18b8e3f163d3b/components/net/http_loader.rs#L617
https://github.com/servo/servo/blob/75c7712905c522f604a454dcb1c18b8e3f163d3b/components/net/http_loader.rs#L641

Reply:
You're right! I've modified this as well as all of the other uses of `CrossProcessInstant` and `Duration` in the DevTools code.